pkg/docker/build.go

```diff
@@ -95,6 +95,7 @@ func BuildAddLabelsAndSchemaToImage(image string, labels map[string]string, bund
 	// We're not using context, but Docker requires we pass a context
 	args = append(args, ".")
 	cmd := exec.Command("docker", args...)
+	cmd.Dir = dir
```

Comment:
Why is dir needed here?

Reply:
it just resolved my issue...

Reply:
Just to be sure I understand you correctly: Which issue are you referring to? Was it related to the `--load` issue, or something else?